bin/md5_checksum.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import sys
+import hashlib
+
+from utils import json_dump, package_validation_dict
+
+
+def md5_checksum_test(file_path: str) -> str:
+    """
+    1. Opens the downloaded file and calculates MD5 Checksum from file contents
+    2. Compares calculated MD5 Checksum to the one provided in the original input csv file
+    3. Returns result of MD5 Checksum test ("pass" or "fail")
+
+    Args:
+        file_path (str): Path to the downloaded file
+
+    Returns:
+        str: Result of MD5 Checksum test
+    """
+    with open(file_path, "rb") as file_to_check:
+        data = file_to_check.read()
+        md5_generated = hashlib.md5(data).hexdigest()
```

Comment:
You should use the version I have [here](https://github.com/Sage-Bionetworks-Workflows/nf-synindex/blob/f14acba8c1588b911fb443a3306f58fb90616aa1/bin/synindex.py#L30-L36), which iterates over the file contents instead of loading it all into memory. I think this is a must since some of these files will be GBs in size, so we would prefer keeping memory usage more or less constant if we can.